packages/core/src/base/staticClient.ts

```diff
@@ -139,8 +159,12 @@ async function runFetch(
     fetchOptions.cacheManager = options.client.clientConfig.cacheManager;
   }
 
+  sdkLogger.info(formatFetchForInfoLog(resource, fetchOptions));
+
   const response = await fetch(resource, fetchOptions);
 
+  sdkLogger.info(formatResponseForInfoLog(response));
```

Comment:
Do we have a test that ensures that this is called on a failure?

Reply:
We have tests that ensure a return from this function on failures like 404's and go down this path.